torchgeo/transforms/indices.py

```diff
@@ -147,21 +147,34 @@ def __init__(self, index_red: int, index_nir: int) -> None:
             index_nir: index of the Near Infrared (NIR) band in the image
         """
         super().__init__(index_a=index_red, index_b=index_nir)
-
-
+        
 class AppendNDWI(AppendNormalizedDifferenceIndex):
     """Normalized Difference Water Index (NDWI).
-
     If you use this index in your research, please cite the following paper:
-
     * https://doi.org/10.1080/01431169608948714
     """
 
     def __init__(self, index_green: int, index_nir: int) -> None:
         """Initialize a new transform instance.
-
         Args:
             index_green: index of the Green band in the image
             index_nir: index of the Near Infrared (NIR) band in the image
         """
         super().__init__(index_a=index_green, index_b=index_nir)
+
+class AppendSWI(AppendNormalizedDifferenceIndex):
+    """Normalized Difference Water Index (NDWI).
+
+    If you use this index in your research, please cite the following paper:
+
+    * https://ui.adsabs.harvard.edu/abs/2020ISPAn.5.3...33J/abstract#:~:text=Sentinel%2D2%20is%20a%20new,higher%20spatial%20and%20spectral%20resolution.&text=The%20results%20showed%20that%20overall,water%20for%20Sentinel%2D2%20data.
```

Comment:
Is this the correct citation? The format of the title is very strange. I also found this paper that might be more applicable -- https://www.mdpi.com/2073-4441/13/12/1647/htm?

Reply:
citation corrected.
https://www.droughtmanagement.info/literature/ISPRS_Various_Drought_Indices_Monitoring_Drought_Condition_Aravalli_Terrain_India_2004.pdf ..
This is the actual paper written by C. Bhuiyan
Department of Civil Engineering, Indian Institute of Technology Kanpur
Kanpur-208016, India, bhuiyan@iitk.ac.in

Reply:
True ..i reviewed and found that  https://www.mdpi.com/2073-4441/13/12/1647/html is more applicable..
i will post a PR changing it asap ..thanks for the guidance.

Reply:
Pushed it with the correct citation https://www.mdpi.com/2073-4441/13/12/1647/html